From: Jeffrey Hugo <jeffrey.l.hugo@gmail.com>
Date: Thu, 28 May 2020 07:51:35 -0700
Subject: arm64: dts: qcom: msm8998-clamshell: Fix label on l15 regulator
Git-commit: 5c29f9e6e7d42e113bd51aeb92a28bf1ae76911c
Patch-mainline: v5.9-rc1
References: git-fixes

The label on the l15 regulator node does not follow the style of the
rest of the regulator nodes.  Luckily, no one has used the label yet,
so lets fix it.

Fixes: 2c6d2d3a580a ("arm64: dts: qcom: Add Lenovo Miix 630")
Signed-off-by: Jeffrey Hugo <jeffrey.l.hugo@gmail.com>
Link: https://lore.kernel.org/r/20200528145135.44269-1-jeffrey.l.hugo@gmail.com
Signed-off-by: Bjorn Andersson <bjorn.andersson@linaro.org>
Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 arch/arm64/boot/dts/qcom/msm8998-clamshell.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/qcom/msm8998-clamshell.dtsi
+++ b/arch/arm64/boot/dts/qcom/msm8998-clamshell.dtsi
@@ -170,7 +170,7 @@
 			regulator-min-microvolt = <1880000>;
 			regulator-max-microvolt = <1880000>;
 		};
-		vreg_15a_1p8: l15 {
+		vreg_l15a_1p8: l15 {
 			regulator-min-microvolt = <1800000>;
 			regulator-max-microvolt = <1800000>;
 		};
